CitizenAPI/src/main/java/com/bae/citizen/rest/CitizenController.java

```diff
@@ -0,0 +1,53 @@
+package com.bae.citizen.rest;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.bae.citizen.domain.Citizen;
+import com.bae.citizen.service.CitizenService;
+
+@RestController
+public class CitizenController {
+
+	private CitizenService citizenService;
+
+	public CitizenController() {
+
+	}
+
+	@Autowired
+	public CitizenController(CitizenService citizenService) {
+
+		this.citizenService = citizenService;
+	}
+
+	@GetMapping("/getCitizensByForenames/{username}/{forenames}")
```

Comment:
What purpose does the '/username' server?

Reply:
Match the endpoint in core. We only want to allow access to logged in users.